src/uct/ib/mlx5/dv/ib_mlx5dv_md.c

```diff
@@ -989,20 +982,20 @@ static ucs_status_t uct_ib_mlx5_devx_md_open(struct ibv_device *ibv_device,
         goto err;
     }
 
-    md = ucs_calloc(1, sizeof(*md), "ib_mlx5_md");
+    md = ucs_derived_of(uct_ib_md_alloc(sizeof(*md), "ib_mlx5_devx_md", ctx),
+                        uct_ib_mlx5_md_t);
     if (md == NULL) {
         status = UCS_ERR_NO_MEMORY;
         goto err_free_context;
     }
 
-    dev                 = &md->super.dev;
-    dev->ibv_context    = ctx;
-    md->mkey_tag        = 0;
+    dev          = &md->super.dev;
+    md->mkey_tag = 0;
     uct_ib_mlx5_devx_mr_lru_init(md);
 
     status = uct_ib_device_query(dev, ibv_device);
     if (status != UCS_OK) {
-        goto err_free;
+        goto err_md_free;
```

Comment:
need to rename this label to err_lru_cleanup

Reply:
done